crates/signer/src/manager.rs

```diff
@@ -117,3 +131,90 @@ impl SigningManager {
         Ok(signer.delegation)
     }
 }
+
+// TODO(David): Add more tests.
```

Comment:
I might come up with something more sensible in terms of tests, but this isn't a blocker for now. Leaving this in for now, if you can think of anything worth testing here.

Reply:
Resolved as a later concern.